src/lexer.d

```diff
@@ -2144,7 +2138,8 @@ public:
                 break;
             }
         }
-        if (errno == ERANGE)
+        const isLong = (result == TOKfloat80v || result == TOKimaginary80v);
+        if (isOutOfRange && !isLong)
```

Comment:
This prepares a bugfix wrt. this ignoring of out-of-range real literals. I can't just error out, as disabled code blocks are still lexed - e.g., unit tests specifically only enabled for x87 reals with huge literals.